js/ui/tool/select.js

```diff
@@ -1,3 +1,7 @@
+/**
+ * TODO: REFACTOR THIS WHOLE THING
+ */
+
```

Comment:
lmao

Reply:
Oh, that was before the latest versions of the new select. We can just remove it now.